serverjs/updatecards.js

```diff
@@ -463,10 +539,10 @@ var specialCaseCardsList = {
     tokenId: "e6fa7d35-9a7a-40fc-9b97-b479fc157ab0"
   }],
   "Westvale Abbey": [{
-    tokenId: " 94ed2eca-1579-411d-af6f-c7359c65de30",
+    tokenId: "94ed2eca-1579-411d-af6f-c7359c65de30",
   }]
 };
-var specialCaseTokensList = ["Food"];
+var specialCaseTokensList = ["Food", "Treasure"];
 
 function getTokensForSpecialCaseCard(newCardid, card) {
```

Comment:
This function could just be defined as an object and could also be moved into a separate json file.